src/components/Main/Schedule/Home.tsx

```diff
@@ -119,6 +88,37 @@ export const Home = () => {
       )}
     </>
   );
+  function onTabClick(e: React.ChangeEvent, value: number) {
```

Comment:
so in the few react apps I've been working on, I've noticed that instead of handling the `(e: React.ChangeEvent, value: number)` all the way at the top where at `ScheduleTabs` the function in tabs is very simple: `onChange={onTabClick}`, and then this gets passed all the way back up to `Home.tsx`, to make it simpler at the container level, and more specific at the functional component level. I don't know the exact reason why, but my guess is that in container level, it's annoying to read so much, and it's simpler when the functional components is given x --> returns y, instead of given x --> returns (y, z)

Reply:
I can make the updates myself to show what I'm talking about, but let's keep this as a rule for when we develop so that we can be consistent between the 3 of us